### ISSUE TITLE
Laravel-graphql + Docker

### ISSUE THREAD

**Issue by DeroSavage** [Folkloreatelier/laravel-graphql#325]
Hey everybody,

i run a laravel application inside docker with the laravel-graphql installed. This seems to be very slow as it takes like 2-3 seconds for a normal query. When i start the laravel application on local its like instantly. I think it could have to do something with graphql because the normal api request like on / which just returns a json array is very fast on both, docker and local. Maybe someone has the same problem with running graphql in docker or maybe its a completely different problem..

**Comment by kevinvdburgt** [comment]
Are you running docker on macos?

**Comment by DeroSavage** [comment]
Yes. Update on this: its way faster now cause i used the php alpine package. So seems that this is a macos docker problem..but if somebody still knows some configuration to make this fast as possible please tell me :D

**Comment by ivanfilatov** [comment]
I was facing same problem, but within swarm (1 manager 2 workers, all linux). I managed to solve it by setting "nobarriers" option to /etc/fstab in ext4 partition. Maybe that helps.
